hazelcast/src/test/java/com/hazelcast/client/cache/ClientCacheCreationTest.java

```diff
@@ -120,7 +121,15 @@ public void recreateCacheOnRestartedCluster_whenMaxConcurrentInvocationLow() {
         HazelcastClientCachingProvider cachingProvider = createClientCachingProvider(client);
         final CacheManager cacheManager = cachingProvider.getCacheManager();
         MutableConfiguration configuration = new MutableConfiguration();
-        cacheManager.createCache("xmlCache", configuration);
+        // ensure cache is created despite the low concurrent invocation limit
+        while (true) {
```

Comment:
Can you convert it to assertTrueEventually to avoid possibility of infinite wait? (practically can not be more than the test timeout anyway)

Reply:
done